ui/app/templates/components/scale-events-chart.hbs

```diff
@@ -1,13 +1,21 @@
 <LineChart
   @timeseries={{true}}
-  @curve="stepAfter"
   @xProp="time"
   @yProp="count"
-  @data={{this.data}}
-  @annotations={{this.annotations}}
-  @activeAnnotation={{this.activeEvent}}
-  @annotationKey="event.uid"
-  @onAnnotationClick={{action this.toggleEvent}} />
+  @data={{this.data}}>
+  <:svg as |c|>
+    <c.Area
+      @curve="stepAfter"
+      @data={{this.data}} />
+  </:svg>
+  <:after as |c|>
+    <c.VAnnotations
+      @annotations={{this.annotations}}
+      @key="event.uid"
+      @activeAnnotation={{this.activeEvent}}
+      @annotationClick={{action this.toggleEvent}} />
+  </:after>
```

Comment:
really dig this - I've not used named yields with the block form yet - it's nice

Reply:
Yeah! I enjoyed how natural it felt given how yields and contextual components already work.